src/pages/home/report/ReportActionItemFragment.js

```diff
@@ -63,6 +64,9 @@ const propTypes = {
     /** Whether the comment is a thread parent message/the first message in a thread */
     isThreadParentMessage: PropTypes.bool,
 
+    /** The report's action name/type e.g. APPROVED, SUBMITTED, etc. */
+    actionName: PropTypes.string,
```

Comment:
I think we can just probably pass `isApprovedOrSubmittedReportActionType` in `ReportActionItemFragment` so that we don't have to recheck it here.

Reply:
Made the change here 1d698f3d0a362bd67fd46ee48fcbd37c14a8004f.

Reply:
https://github.com/Expensify/App/pull/30193/files/64d204b196a8295ad037b97a967702cea6245d17#diff-7451957e0135ee20216b214d4e27202271aeacbd2e321b9b35b5b1e60d6677b6